Tao_Current.txt

```diff
@@ -329,16 +382,17 @@ Another aspect of Working Groups that confounds many people is the fact that the
 
 The lack of formal voting has caused some very long delays for some proposals, but most IETF participants who have witnessed rough consensus after acrimonious debates feel that the delays often result in better protocols. (And, if you think about it, how could you have "voting" in a group that invites all interested individuals to participate, and when it's impossible to count the participants?) Rough consensus has been defined in many ways; a simple version is that it means that strongly held objections must be debated until most people are satisfied that these objections are wrong.
 
-A related problem is that some people think that their proposals should be discussed in the WG even when the WG Chairs do not. For example, if the proposed work is not really part of the charter, the WG Chairs may constrain the discussion of the proposal in order to keep the WG focused on the chartered work. Individuals who think that a WG should bring up a topic that is considered off-charter by the WG Chairs can bring their concerns to the responsible AD; the AD may agree to add the topic to the charter, or that it is already covered in the charter, or that the WG Chairs are correct and that the participant should work on the proposal outside the WG.
+A related problem is that some people think that their proposals should be discussed in the WG even when the WG Chairs do not. For example, if the proposed work is not really part of the charter, the WG Chairs may constrain the discussion of the proposal in order to keep the WG focused on the chartered work. Individuals who think that a WG should bring up a topic that is considered off-charter by the WG Chairs can bring their concerns to the responsible AD; the AD may agree to adjust the charter to add the topic with the working group, or that it is already covered in the charter, or that the WG Chairs are correct and that the participant should work on the proposal outside the WG.
```

Comment:
It seems like in practice these requests to adjust the charter come from the chairs.